plugins/azure-functions/src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react';
+import Alert from '@material-ui/lab/Alert';
+
+interface Props {}
+interface MyProps {}
+
+interface MyState {
+  hasError: boolean;
+}
+
+export default class ErrorBoundary extends Component<MyProps, MyState> {
```

Comment:
I'm pretty sure you don't need to provide this as Backstage will handle this for you. Generally if you have errors you can use the `<ResponseErrorPanel>` component for that